### ISSUE TITLE
Breadcrumb name not properly visible at 200% zoom

### ISSUE THREAD

**Issue by mharnetiaux** [microsoft/fluentui#17653]
### Environment Information

- All Browsers

### Describe the issue:

This was called out in a HCL review.

1. Open a example Breadcrumb in Fluentui.
3. Open dev tools and dock inspector to right side.
4. Drag screen below 360px.
5. Increase zoom level to 200% in Browser.
6. Update a Breadcrumb item with long text, such as "Diversity inclusion and being an ally across cultures"
7.  Notice dialog is not properly visible.

#### Actual behavior:

Breadcrumb item is not properly visible on small screens with long names at zoom level 200%.

#### Expected behavior:

Breadcrumb item is properly visible on small screens with long names at zoom level 200%.

**Comment by SerinaJingyiLu** [comment]
I met the same problem. Which version do you use?

**Comment by tobybaratta** [comment]
I definitely don't want to be a pain on this one, but will it not need to actually support 400% due to WCAG 2.1.10 (reflow)? I'm a bit worried if 200% failing.

**Comment by gouttierre** [comment]
@mharnetiaux - thanks for reaching out. 

@miroslavstastny - Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @smhigley

**Comment by miroslavstastny** [comment]
The same root-cause as in https://github.com/microsoft/fluentui/issues/17652#issuecomment-860710761.

**Comment by ecraig12345** [comment]
I think this should be closed as a duplicate of #17652 since it's the same root cause.
